project/README.md

```diff
@@ -0,0 +1,53 @@
+# Basic information
+GitHub repository: https://github.com/haashiagh10/propsal
+
+# Problem to Solve
+Raman Spectroscopy , and High-Performance Liquid Chromatography(HPLC) are the way to get Raman spectroscpy data.To deal with the data , We want to detect the feature in the liqid by checking the Raman data.
+However , the machine to get the spectral data and the model background are not quite clear.
+After remove the noise and background in the data , we can implement some algorithm to analyze the data.
+
+# System Architecture
+The module should be compatible with NumPy arrays
+![image](https://github.com/haashiagh10/propsal/blob/main/flowchart.png)
+
+# API Description
+Ideally, it would be available on PyPI, and one can install it using pip.
```

Comment:
It is not necessary to make it in PyPI.  It's not very straight-forward to do it with a project using C++.

Reply:
Sorry professor , what means add to the list in project root directory. I have pull request to the project folder on github. But I cannot see the folder in project folder. Do i miss something?